package.json

```
@@ -27,7 +27,9 @@
     "compile": "run-s clean:lib clean:modules build:types babel:lib babel:modules",
     "prepublishOnly": "run-s compile deploy",
     "prepare": "yarn --cwd react --ignore-scripts",
-    "postreleasy": "vtex publish --public --verbose"
+    "postreleasy": "vtex publish --public --verbose",
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook"
```

just to maintain the standard already used
```suggestion
    "build:storybook": "build-storybook"
```